timber-lint/src/main/java/timber/lint/WrongTimberUsageDetector.java

```diff
@@ -523,4 +549,8 @@ private static boolean checkConditionalUsage(JavaContext context, PsiMethodCallE
       Issue.create("TimberTagLength", "Too Long Log Tags", "Log tags are only allowed to be at most"
               + " 23 tag characters long.", Category.CORRECTNESS, 5, Severity.ERROR,
           new Implementation(WrongTimberUsageDetector.class, Scope.JAVA_FILE_SCOPE));
+  public static final Issue ISSUE_EXCEPTION_LOGGING =
+      Issue.create("TimberExceptionLogging", "Exception Logging", "Exceptions only should be logged"
+              + " in a certain way.", Category.CORRECTNESS, 5, Severity.WARNING,
```

Comment:
"Explicitly including the exception message is redundant when supplying an exception to log."